terraform-aws-github-runner/modules/runners/lambdas/runners/src/scale-runners/gh-auth.ts

```diff
@@ -32,14 +32,16 @@ async function getCredentialsFromSecretsManager(
 ): Promise<GithubCredentials> {
   return await locallyCached('secretCache', secretsManagerSecretsId, 10 * 60, async () => {
     try {
-      const secretsManager = new SecretsManager();
+      const secretsManager = new SecretsManager({
+        region: Config.Instance.awsRegion,
+      });
 
       const data = await expBackOff(() => {
         return metrics.trackRequest(
           metrics.smGetSecretValueAWSCallSuccess,
           metrics.smGetSecretValueAWSCallFailure,
           () => {
-            return secretsManager.getSecretValue({ SecretId: secretsManagerSecretsId }).promise();
+            return secretsManager.getSecretValue({ SecretId: secretsManagerSecretsId });
```

Comment:
seems promise is still supported in the latest version.

https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/Request.html#promise-property

Is there a reason you migrated from a async call to a sync call?

Reply:
The auto converter removed those.  As per the migration guide, it seems like it automatically returns a promise without it needing to be explicitly referenced:
https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/migrating.html